docs/README.md

```diff
@@ -0,0 +1,48 @@
+# kcp Documentation Setup
+
+## Overview
+
+Our documentation is powered by [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/) with some 
+additional plugins and tools:
+
+- [awesome-pages plugin](https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin)
+- [macros plugin](https://mkdocs-macros-plugin.readthedocs.io/en/latest/)
+- [mike](https://github.com/jimporter/mike) for multiple version support
+
+We have support in place for multiple languages (i18n), although we currently only have documentation in English. If 
+you're interested in contributing translations, please let us know!
+
+## File structure
+
+All documentation-related items live in `docs` (with the small exception of various `make` targets and some helper 
+scripts in `hack`).
+
+The structure of `docs` is as follows:
+
+| Path                         | Description                                                                       |
+|------------------------------|-----------------------------------------------------------------------------------|
+| config/<language>/mkdocs.yml | Language-specific `mkdocs` configuration.                                         |
```

Comment:
This omitted the language in GH preview. `config//mkdocs.yml` is what I see.

Reply:
Same with `content/`
<img width="600" alt="Screen Shot 2023-02-16 at 4 02 41 PM" src="https://user-images.githubusercontent.com/321387/219486224-d4be61cc-2512-4b67-9af6-b394dcfd670b.png">

Reply:
thx fixing